### PR TITLE
ops: avoid mangling org and net metrics

### PR DESCRIPTION
**Pull request by alexgarel**
by setting a different influxdb database (they use same instance)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ x-robotoff-base-env: &robotoff-base-env
   OFF_PASSWORD:
   INFLUXDB_HOST:
   INFLUXDB_PORT:
+  INFLUX_DB_NAME:
```
Comment (ocervell) [inline docker-compose.yml:31]: This should be `INFLUXDB_NAME`

File: .github/workflows/container-deploy.yml [hunk 2]
```diff
@@ -101,7 +101,7 @@ jobs:
           echo "TF_SERVING_HOST=${{ secrets.TF_SERVING_HOST }}" >> .env
           echo "INFLUXDB_HOST=${{ secrets.INFLUXDB_HOST }}" >> .env
           echo "INFLUXDB_PORT=8086" >> .env
-          echo "INFLUXDB_NAME=db0" >> .env
+          echo "INFLUXDB_NAME=${{ env == 'robotoff-org' && 'off_metrics' || 'off_net_metrics' }}" >> .env
```
Comment (ocervell) [inline .github/workflows/container-deploy.yml:104]: I'm not sure `env == ` will work here, but `matrix.env ==` should ?

**Review by ocervell — CHANGES_REQUESTED** [review]

File: .github/workflows/container-deploy.yml (hunk 2 above)
Comment (alexgarel) [reply .github/workflows/container-deploy.yml:104]: thanks !

File: docker-compose.yml (hunk 1 above)
Comment (alexgarel) [reply docker-compose.yml:31]: seems I was tired while writing this :sweat_smile:

**Comment by alexgarel** [comment]
@ocervell if you want to review again ! Thanks…

**Review by ocervell — APPROVED** [review]
LGTM

**Pull request merged**